titiler/stacapi/settings.py

```diff
@@ -77,6 +77,7 @@ class STACAPISettings(BaseSettings):
     """STAC API settings"""
 
     stac_api_url: str
+    mspc_default_api_url: str = "https://planetarycomputer.microsoft.com/api/stac/v1"
```

Comment:
I think it's fine having `stac_api_url` to default to `https://planetarycomputer.microsoft.com/api/stac/v1`